### PR TITLE
code review

### PR DESCRIPTION
**Pull request by ely-alamillo**

### CODE REVIEW

**Comment by mixelpixel** [comment]
just adding a comment to make sure I get included in the PR emails

**Comment by SunJieMing** [comment]
Good job committing often

File: .gitIgnore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+data
+.DS_Store
+node_modules
```
Comment (SunJieMing) [inline .gitIgnore]: Thank you so much for doing this

File: models.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+
+const UserSchema = new mongoose.Schema({
+  username: {
```
Comment (SunJieMing) [inline models.js]: `username` should be `unique` and `required`.
```
username: {
  type: String,
  required: true,
  unique: true
}
```

File: models.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  },
+});
+
+const User = mongoose.model('User', UserSchema);
+const Blog = mongoose.model('Blog', PostSchema);
+
+module.exports = {
+  User,
+  Blog,
+}
```
Comment (SunJieMing) [inline models.js]: Put a semicolon here

File: models.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  username: {
+    type: String,
+  },
+  createAt: {
+    type: Date,
+    default: Date.now,
+  },
+});
+const PostSchema = new mongoose.Schema({
+  title: {
```
Comment (SunJieMing) [inline models.js]: Some of these props should be `required`

File: .gitIgnore (hunk 1 above)
Comment (mixelpixel) [reply .gitIgnore]: 👍

**Comment by mixelpixel** [comment]
Thanks Ben!

**Pull request closed without merging**